src/lib/Form/EventListener/BuildPathFromRootListener.php

```diff
@@ -41,6 +41,9 @@ public function onPreSubmitData(FormEvent $event)
         $data = $event->getData();
         if (!array_key_exists('site_root', $data) || false === (bool)$data['site_root']) {
             $location = $this->locationService->loadLocation($data['location']);
+            if ($location->depth <= 1) {
```

Comment:
We mostly use Yoda condition

Reply:
> We mostly use Yoda condition

But that's optional, we don't force it. Here what we have looks more readable to me than Yoda :P

Reply:
Of course, it is only social contract ;)